MakingCoffee/src/MakingCoffeeEx1.java

```diff
@@ -0,0 +1,43 @@
+
+import java.util.Scanner;
+public class MakingCoffeeEx1 {
+
+           public static void main(String[] args) {
+            int waterPerCup = 200;
+            int milkPerCup = 50;
+            int coffeeBeansPerCup = 15;
+            int cupsOfCoffee1 = 2;
+               int cupsOfCoffee2 = 5;
+
+            System.out.println("Write how many ml of water the coffee machine has: ");
+               System.out.print("> ");
+            Scanner value = new Scanner (System.in);
+            int waterValue = value.nextInt();
+
+            System.out.println("Write how many ml of milk the coffee machine has: ");
+               System.out.print("> ");
+            int milkValue = value.nextInt();
+
+            System.out.println("Write how many grams of coffee beans the coffee machine has: ");
+               System.out.print("> ");
+            int coffeeBeansValue = value.nextInt();
+
+            System.out.println("Write how many cups of coffee you will need: ");
+               System.out.print("> ");
+            int neededCupsOfCoffee = value.nextInt();
+
+            if (waterValue % waterPerCup >= neededCupsOfCoffee && milkValue % milkPerCup >= neededCupsOfCoffee && coffeeBeansValue % coffeeBeansPerCup >= neededCupsOfCoffee)
+                System.out.println("Yes, I can make that amount of coffee");
```

Comment:
Try to do following:
1. Calculate how many caps of coffee can make coffee machine with the available resources of milk, water and coffee.
2. Calculate difference between that number and `neededCupsOfCoffee`.
3.Compare them, if it is less then `neededCupsOfCoffee` , program should print: No, I can make only * cup(s) of coffee 
when greater then:  "Yes, I can make that amount of coffee (and even * more than that)".
when equals: "Yes, I can make that amount of coffee".